osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/hibernate/InetAddressUserType.java

```diff
@@ -31,13 +31,14 @@
         return new int[] { TextType.INSTANCE.sqlType() };
     }
 
+    @SuppressWarnings("rawtypes")
```

Comment:
Consider using 
`public Class<?> returnedClass()`
in stead of suppressing the warning